src/main/java/com/google/devtools/build/lib/sandbox/SandboxHelpers.java

```diff
@@ -231,12 +233,12 @@ public static void populateInputsAndDirsToCreate(
 
     // And all parent directories of output files. Note that we don't add the files themselves --
     // any pre-existing files that have the same path as an output should get deleted.
-    for (PathFragment file : outputFiles) {
+    for (PathFragment file : outputs.files().values()) {
       dirsToCreate.add(file.getParentDirectory());
     }
 
     // Add all output directories.
-    dirsToCreate.addAll(outputDirs);
+    dirsToCreate.addAll(outputs.dirs().values());
```

Comment:
Let's add a couple of tests in `SandboxHelpersTest.java`:

1. Verify that `populateInputsAndDirsToCreate` collects the right set of directories when path mapping is enabled.
2. Verify that `moveOutputs` moves from/to the right location when path mapping is enabled.

Reply:
Done!